crates/bevy_render/src/texture/image.rs

```diff
@@ -186,7 +186,9 @@ impl Default for Image {
                 label: None,
                 mip_level_count: 1,
                 sample_count: 1,
-                usage: wgpu::TextureUsages::TEXTURE_BINDING | wgpu::TextureUsages::COPY_DST,
+                usage: wgpu::TextureUsages::TEXTURE_BINDING
+                    | wgpu::TextureUsages::COPY_DST
+                    | wgpu::TextureUsages::COPY_SRC,
```

Comment:
I'm not aware of any problems of adding COPY_SRC generally, but I do wonder if there is any.

Reply:
Afaik we shouldn't be adding usages we don't need, it'll hurt perf. I'd like to avoid this. Same with COPY_DST.